Integrations/integration-VirusTotal-Private_API.yml

```diff
@@ -87,6 +101,33 @@ script:
     ''' HELPER FUNCTIONS '''
 
 
+    def is_enough_preferred_vendors(scan_results):
+        if not (PREFERRED_VENDORS and PREFERRED_VENDORS_THRESHOLD):
+            return False
+        if PREFERRED_VENDORS and not PREFERRED_VENDORS_THRESHOLD:
+            return_error("Error: If you entered Preferred Vendors you must also enter Preferred Vendors Threshold")
+        if "scans" not in scan_results:
+            return False
+
+        counter = 0
+
+        vendors_scans_dict = scan_results["scans"]
+        list_of_preferred_vendors = PREFERRED_VENDORS.split(',')
+
+        for i in range(len(list_of_preferred_vendors)):
+            list_of_preferred_vendors[i] = list_of_preferred_vendors[i].lower()
+
+        for vendor_name in vendors_scans_dict:
+            cur_vendor_scan = vendors_scans_dict[vendor_name]
+            vendor_name_in_lowercase = vendor_name.lower()
+
+            if vendor_name_in_lowercase in list_of_preferred_vendors:
+                if "detected" in cur_vendor_scan and cur_vendor_scan["detected"]:
+                    counter += 1
+
+        return int(PREFERRED_VENDORS_THRESHOLD) <= counter
```

Comment:
Check during `test-instance`: `int(PREFERRED_VENDORS_THRESHOLD)` does not throw exception.